include/ring-core/target.h

```diff
@@ -50,6 +50,9 @@
 #elif defined(__riscv) && __SIZEOF_POINTER__ == 8
 #define OPENSSL_64_BIT
 #define OPENSSL_RISCV64
+#elif defined(__riscv) && __SIZEOF_POINTER__ == 4
+#define OPENSSL_32_BIT
+#define OPENSSL_RISCV32
```

Comment:
This is what BoringSSL has:
```
#elif defined(__riscv) && __SIZEOF_POINTER__ == 8
#define OPENSSL_64_BIT
#define OPENSSL_RISCV64
#elif defined(__riscv) && __SIZEOF_POINTER__ == 4
#define OPENSSL_32_BIT
```
In particular, they don't define `OPENSSL_RISCV32`. So I suggest we remove that. My plan is to remove all of the `OPENSSL_{arch}` defines for architectures where we do not have architecture-specific C/asm code, including `OPENSSL_RISCV64` above, in the near future.

Reply:
Agreed, I have remove the "OPENSSL_RISCV32".